### PR TITLE
Add exec kind to annotation for an action and show it in action list.

### LINKED ISSUE #427: add ability to distinguish plain actions from sequences at the "list" level
From @starpit : 
right now, the `wsk action list` output doesn't not allow me to distinguish sequences from actions, at least for sequences created on the CLI

for sequences created in the UI, i have been adding an annotation `{ key: "isSequence", value: true }`

it'd be great if we could update the CLI to adopt this behavior. otherwise, i cannot know how to render an action, without doing a "get" on it.

### PR DESCRIPTION
**Pull request by rabbah**
Closes #427.

### CODE REVIEW

File: tools/cli/go-whisk-cli/commands/util.go [hunk 1]
```diff
@@ -263,7 +263,8 @@ func printActionList(actions []whisk.Action) {
         if action.Publish {
             publishState = wski18n.T("shared")
         }
-        fmt.Printf("%-70s %s\n", fmt.Sprintf("/%s/%s", action.Namespace, action.Name), publishState)
```
Comment (rabbah) [inline tools/cli/go-whisk-cli/commands/util.go]: @dubeejw if getValueString doesn't find the `exec` property, what happens?

Comment (dubeejw) [inline tools/cli/go-whisk-cli/commands/util.go]: Yes, everything should be fine if exec isn't present.

**Comment by rabbah** [comment]
pg2/560 approved.

**Comment by psuter** [comment]
Squashed and merged as df8d99a177e06070e74ed4e0524ee5083c1a80ae

**Pull request closed without merging**